junifer/data/masks.py

```diff
@@ -28,6 +45,12 @@
 _available_masks: Dict[str, Dict[str, Any]] = {
     "GM_prob0.2": {"family": "Vickery-Patil"},
     "GM_prob0.2_cortex": {"family": "Vickery-Patil"},
+    "compute_brain": {"family": "Callable", "func": compute_brain_mask},
```

Comment:
I think it would be better to have the actual function name from nilearn (i.e. "compute_brain_mask") rather than an abbreviation of it.